src/main/java/uk/gov/hmcts/reform/sscs/callback/handlers/IssueFurtherEvidenceHandler.java

```diff
@@ -80,6 +81,11 @@ private void doIssuePerDocumentType(SscsCaseData caseData, List<FurtherEvidenceL
 
     private void postIssueFurtherEvidenceTasks(SscsCaseData caseData) {
         try {
+            if (caseData.getReasonableAdjustmentsLetters() != null) {
+                final SscsCaseDetails sscsCaseDetails = ccdService.getByCaseId(Long.valueOf(caseData.getCcdCaseId()), idamService.getIdamTokens());
+                caseData = sscsCaseDetails.getData();
+            }
+
             setEvidenceIssuedFlagToYes(caseData.getSscsDocument());
             ccdService.updateCase(caseData, Long.valueOf(caseData.getCcdCaseId()),
```

Comment:
Is the event comment here still relevant? Abit confusing as the evidence hasn't actually been issued. Think we might need to change it to something related to reasonable adjustments.